package.json

```diff
@@ -16,6 +16,7 @@
   "scripts": {
     "prepare": "npm run prepare:sass && npm run prepare:fonts",
     "prepare:sass": "sass style.scss core-style.css --load-path node_modules/@govflanders",
+    "prepare:test:components": "cp -R ./test/e2e/components/vl-element.js ../../test/e2e/components/vl-element.js", 
```

Comment:
Dit zou niet nodig moeten zijn wanneer de ES6 imports of Node imports gebruikt worden.

Reply:
Fixed